modules/canbus/vehicle/BUILD

```diff
@@ -47,6 +47,7 @@ cc_library(
     deps = [
         "//modules/canbus/vehicle/lincoln:lincoln_vehicle_factory",
         "//modules/canbus/vehicle/gem:gem_vehicle_factory",
+	"//modules/canbus/vehicle/ch:ch_vehicle_factory",
```

Comment:
Align.

Reply:
The code is already aligned, please review again, think you!